.circleci/config.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
```

Comment:
Circle CI was chosen because of the [speculation](https://twitter.com/alicegoldfuss/status/1098604563664420865) around the future of Travis CI.